src/context/reducers/notificationReducer.tsx

```diff
@@ -10,40 +10,44 @@ export enum NotificationItemVariant {
 }
 
 export interface NotificationItem {
-  variant: 'success' | 'danger' | 'warning' | 'info' | 'default'
-  title: string;
-  message: string;
-  timestamp: Moment;
+  variant: 'success' | 'danger' | 'warning' | 'info' | 'default', //here need? how does it work? what's it doing? remove?
+  title: string,
+  message: string,
+  timestamp: Moment
 }
 
 export type NotificationState = NotificationItem[];
 
 export const initialNotificationsState: NotificationState = [];
-
-// { 'variant': 'info', 'title': 'Unread info notification title', 'message': 'This is an info notification description', 'timestamp': moment().subtract(1, 'days') },
-// { 'variant': 'warning', 'title': 'Unread warning notification title', 'message': 'This is a warning!!', 'timestamp': moment().subtract(5, 'minutes') },
-// { 'variant': 'danger', 'title': 'Unread danger notification title', 'message': 'This is dangerous...', 'timestamp': moment().subtract(1, 'hour') },
-
 interface NotificationPayload {
   [NotificationActionTypes.SEND]: {
     notifications: NotificationItem[]
+  },
+  [NotificationActionTypes.CLEAR]: {},
+  [NotificationActionTypes.REMOVE]: {
+    notifications: NotificationItem[]
   }
 }
 
 export type NotificationActions = ActionMap<NotificationPayload>[
   keyof ActionMap<NotificationPayload>
 ]
 
-
 export const notificationsReducer = (
   state: NotificationState,
   action: NotificationActions
 ) => {
   switch (action.type) {
-    case NotificationActionTypes.SEND: {
-      return state.concat(action.payload.notifications).sort((a, b) => b.timestamp.diff(a.timestamp)) // earliest at the front
-    }
+    case NotificationActionTypes.SEND:
+      return state.concat(action.payload.notifications).sort((a, b) => b.timestamp.diff(a.timestamp)); // chronological order for notifications
```

Comment:
Is there a reason why you used `concat`? I would suggest replacing it with the spread operator to keep the codebase consistent.

```suggestion
      return [...state, ...action.payload.notifications].sort((a, b) => b.timestamp.diff(a.timestamp)); // chronological order for notifications
```

Reply:
No reason why it used concat, legacy code from before that I didn't change